frontend/src/app/testResults/submissions/Submission.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Story } from "@storybook/react";
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import { Provider } from "react-redux";
+import createMockStore from "redux-mock-store";
+
+import { getMocks, StoryGraphQLProvider } from "../../../stories/storyMocks";
+
+import Submission from "./Submission";
+
+const mockStore = createMockStore([]);
+const store = mockStore({
+  organization: {
+    name: "Central Schools",
+  },
+  facilities: [{ id: "1", name: "Lincoln Middle School" }],
+});
+
+export default {
+  title: "Submission",
```

Comment:
should we put this and the Submissions and Uploads storybooks in a sub-folder of the App directory? kinda like [the test queue stories](https://github.com/CDCgov/prime-simplereport/blob/237094f0bc81a8702a84a47ae8f5e22c086b04c8/frontend/src/stories/testQueue/QueueItem.stories.tsx#L13), for example

Reply:
Oh! This does look much better, thanks. Done in [ffd6eef](https://github.com/CDCgov/prime-simplereport/pull/3766/commits/ffd6eef16767cc963125b8998aae13e2bd9fc04a)

<img width="187" alt="Screen Shot 2022-06-15 at 7 19 36 PM" src="https://user-images.githubusercontent.com/27730981/173958142-5551a4b5-35de-41c1-aabf-31a0312f8c30.png">